BitwardenShared/UI/Tools/Generator/Generator/GeneratorProcessorTests.swift

```diff
@@ -13,6 +13,8 @@ class GeneratorProcessorTests: BitwardenTestCase { // swiftlint:disable:this typ
     var generatorRepository: MockGeneratorRepository!
     var pasteboardService: MockPasteboardService!
     var policyService: MockPolicyService!
+    var reviewPromptService: MockReviewPromptService!
+    var stateService: MockStateService!
```

Comment:
⛏️ Looks like state service isn't used here anymore.
```suggestion
    var reviewPromptService: MockReviewPromptService!
```